catboost/python-package/catboost/_catboost.pyx

```diff
@@ -2508,7 +2510,7 @@ cdef class _CatBoost:
     cpdef bool_t _is_groupwise_learned_model(self):
         return IsGroupwiseLearnedModel(dereference(self.__model))
 
-    cpdef _calc_fstr(self, type_name, _PoolBase pool, int thread_count, int verbose):
+    cpdef _calc_fstr(self, type_name, _PoolBase pool, int thread_count, int verbose, shap_mode="Auto"):
```

Comment:
Let's remove default for `shap_mode` here. It is better to set default value in a single place. I.e. only in `core.py`